lib/grandpa/message_handler.go

```diff
@@ -458,6 +522,14 @@ func (s *Service) VerifyBlockJustification(hash common.Hash, justification []byt
 		if !ok {
 			return ErrInvalidSignature
 		}
+
+		count++
+	}
+
+	// uses the current set of authorities to define the threshold
+	threshold := (2 * len(auths) / 3)
```

Comment:
Thanks! But why the 2 and why the 3? That's more what I was pointing to set as named constants to explain that.

Reply:
`threshold` is two-thirds the number of authorities (@noot, please correct me if I'm wrong). Just 2 and 3 as const doesn't explain what is the porpouse IMO, I can improve the comment.

Reply:
That makes sense. Maybe add a comment above the threshold? Thanks for clarifying it to me 👍

Reply:
yep, the threshold is 2/3 the # of authorities, we need greater than that number of total votes for a block for it to pass (and total votes = # of direct votes for a block or its descendant + # equivocatory voters)